src/universal/subscriptions/subscriber.js

```diff
@@ -21,3 +21,25 @@ export default function subscriber(channel, key, handlers) {
   });
   return () => socket.unsubscribe(channelName);
 }
+
```

Comment:
i'll leave this in for now since we'll eventually use this when we move to paginating the archive.